bitrise.yml

```diff
@@ -0,0 +1,155 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- push_branch: dev
+  workflow: deploy_staging
+- push_branch: master
+  workflow: deploy_appstore
+- push_branch: "*"
+  workflow: test
+workflows:
+  deploy_staging:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@6.0: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - xcode-test@2: {}
+    - fastlane-match@0:
+        inputs:
+        - type: adhoc
+        - team_id: "$TEAM_ID"
+        - app_id: "$BUNDLE_ID"
```

Comment:
`- app_id: "$BUNDLE_ID_STAGING"`

staging and production apps have different bundle ids

Reply:
The $BUNDLE_ID here is not global env, its value depended on the workflow.